README.md

```diff
@@ -0,0 +1,84 @@
+<h1 align="center">
+  <img src="https://avatars1.githubusercontent.com/u/29598503?v=3&s=256" alt="mutt">
+  <br />
+  mutt
+</h1>
+
+<p align="center"><b>This is the snap for mutt</b>, <i>“Mutt is a sophisticated text-based Mail User Agent.”</i>. It works on Ubuntu, Fedora, Debian, and other major Linux
+distributions.</p>
+
+<p align="center">
```

Comment:
Can you add CI badges in here (like [this](https://github.com/snapcrafters/signal-desktop/blob/d581150fdf8e46bbdb1ea8a1fdce278330a8b2ef/README.md?plain=1#L11-L13)) :slightly_smiling_face: